test/e2e/manage-type-external.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+source "${SRC_DIR}/setup-shell.sh"
+
+set -x # echo so that users can understand what is happening
+set -e # exit on error
+
+EXT_CLUSTER_NAME=ext
+
+:
+: -------------------------------------------------------------------------
+: Apply a post-create-hook for testing
```

Comment:
"apply" can be taken two ways: (1) a PCH is a verb, so "apply" means to invoke the verb, or (2) "apply" in the sense of "ensure the thing's definition is _this_". I think you mean the latter. Since, in the context of this script, it is certain that the PCH is not yet defined, I think it would be clearer to write "Define a PCH for testing".

Reply:
ok, updated